mmv1/products/discoveryengine/DataConnector.yaml

```diff
@@ -0,0 +1,331 @@
+                  results.
+                * `description`: The description for data record. This would be displayed
+                  on search results.
+            - name: 'dataStore'
+              type: String
+              description: |
+                The full resource name of the associated data store for the source
+                entity.
+                Format: `projects/*/locations/*/collections/*/dataStores/*`.
+                When the connector is initialized by the DataConnectorService.SetUpDataConnector
+                method, a DataStore is automatically created for each source entity.
+              output: true
+            - name: 'params'
+              type: KeyValuePairs
+              description: |
+                The parameters for the entity to facilitate data ingestion.
+      - name: 'create_time'
+        type: Time
+        description: |
+          Timestamp when the DataConnector was created.
+        output: true
+      - name: 'latest_pause_time'
+        type: Time
+        description: |
+          The most recent timestamp when this [DataConnector][] was paused,
+          affecting all functionalities such as data synchronization.
+          Pausing a connector has the following effects:
+            - All functionalities, including data synchronization, are halted.
+            - Any ongoing data synchronization job will be canceled.
+            - No future data synchronization runs will be scheduled nor can be
+          triggered.
+        output: true
+      - name: 'last_sync_time'
+        type: Time
+        description: |
+          For periodic connectors only, the last time a data sync was completed.
+        output: true
+      - name: 'update_time'
+        type: Time
+        description: |
+          Timestamp when the DataConnector was updated.
+        output: true
+      - name: 'errors'
+        type: Array
+        description: |
+          The errors from initialization or from the latest connector run.
+        output: true
+        item_type:
+          type: NestedObject
+          properties:
+            - name: 'code'
+              type: Integer
+              description: |
+                The status code, which should be an enum value of google.rpc.Code.
+              output: true
+            - name: 'message'
+              type: String
+              description: |
+                A developer-facing error message, which should be in English.
+              output: true
+      - name: 'kms_key_name'
+        type: String
+        description: |
+          The KMS key to be used to protect the DataStores managed by this connector.
+          Must be set for requests that need to comply with CMEK Org Policy
+          protections.
+          If this field is set and processed successfully, the DataStores created by
+          this connector will be protected by the KMS key.
+      - name: 'actionState'
+        type: Enum
```

Comment:
Nit: We normally model output-only enums as Strings, so that it is easier to support additional values without a client-side update. Would apply here and to a few other enums below.